docs/courses/README.md

```diff
@@ -1,28 +1,104 @@
+- [Caution](#caution)
+  - [Languages that don't use the Latin alphabet](#non-latin-alphabet)
+
+## Getting started
+
+- Totally new to LibreLingo? Head to the [Basics](#basics) section! It will give you a general understanding about things.
+- Want to edit or improve existing courses? Check out [this page about editing courses](editing-courses.md).
+- Want to create a new course? Read out [this guide on how to create new courses](creating-courses.md). (Advanced)
+
+
+## Basics
+
+### Terminology
+
+You'll need to have an idea on basic terminology used in LibreLingo:
+
+- A **Course** is a set of Modules. There is typically one course for each language pair. e.g. Spanish for English speakers, Japanese for Italian speakers, etc.
+- A **Module** is a set of Skills.
+- A **Skill** is a small unit of course material. In the web app, the user will typically practice a Skill at a time.
+
+These concepts represent a hierarchical structure to LibreLingo course material: `Courses > Modules > Skills`
+
+- The **target language** is the language the course is meant to teach.
+- The **source langauge** is a language that users of the course are assumed to know.
+
+The course directory name is usually structured in `<destination language>-from-<target audience's language>`. e.g. `french-from-english` teaches French to English speakers.
+
+<a id="exploring-courses"></a>
+### Exploring the courses
+
+To have a better understanding of how courses work, you can explore how the course files are organized on this repository.
+
+On the `courses` directory of [this repository](https://github.com/kantord/LibreLingo/tree/main/courses) you will find courses. Under each course you'll find modules. And inside each module you'll find skills, which are saved as individual .yaml files.
+
+Now let's look into how they look on the frontend, to the user.
+
+If you go to the [homepage](https://librelingo.app/), courses are listed there. e.g. You can see a "START LEARNING SPANISH" button. (**NOTE:** Depending on the state and completeness of the courses, all of them might not be shown there on the website.) So basically, there are courses for each language. You can think of "courses" as "languages" on your mind if that's more comfortable to you.
+
+When you enter a course page, for example the Spanish course [here](https://librelingo.app/course/spanish-from-english/), you'll see headings like Basics, Introduction, Activities. These are modules. And under each module heading there are a list of skills under it. Modules are merely there to serve the purpose of organizing skills and basically nothing else. Think of them as folders for your skills.
+
+"Skills" are the actual lessons focused on a particular topic. Each skill is like a set of questions that follow a particular theme. For example, "Animal" skill has questions relating to animals, "Food" skill has questions relating to foods. Each skill is designed to test the skill on a certain topic.
```

Comment:
```suggestion
"Skills" are units of course content focused on a particular topic. Each skill is a set of new knowledge, such as new words or phrases.  Skills can focus on vocabulary and follow a theme: the "Animal" skill has has words and phrases related animals, the "Food" skill is all about food. Skills can also focus on teaching grammar. When it comes to teaching grammar, your main tool is to teach by example.
```

Reply:
Couple of things:
- '"Animal" skill has has words' > '"Animal" skill has words'
- "phrases related animals" > "phrases related to animals"
- I think "When it comes to teaching grammar, your main tool is to teach by example." is an excellent point but it would be more appropriate on creating courses/skills md files, since this heading is about exploring the course system.

Reply:
I've made the first 2 changes and added a new title "Tips for creating good skills" in `skill.md` and put your advice under it. Feel free to suggest more tips to be added. I think this will be useful for new course designers.